apps/meteor/packages/rocketchat-livechat/assets/demo.html

```diff
@@ -31,15 +31,6 @@ <h2>without changing page title</h2>
 	<a href="#page-5">page 5</a><br>
 	<a href="#page-6">page 6</a><br>
 	<a href="#page-7">page 7</a><br>
-
-	<script type="text/javascript">
-	(function(w, d, s, u) {
-		w.RocketChat = function(c) { w.RocketChat._.push(c) }; w.RocketChat._ = []; w.RocketChat.url = u;
-		var h = d.getElementsByTagName(s)[0], j = d.createElement(s);
-		j.async = true; j.src = `${window.location.origin}/livechat/rocketchat-livechat.min.js?_=201903270000`;
-		h.parentNode.insertBefore(j, h);
-	})(window, document, 'script', `${window.location.origin}/livechat`);
-	</script>
```

Comment:
Why are we removing this?

Reply:
It is a fix that I have to do to the demo page, we have a duplicated livechat install there, but I'll do it on a dedicated PR